setup.py

```diff
@@ -37,4 +37,4 @@
 		'Operating System :: MacOS :: MacOS X'
 	],
 	test_suite='tests'
-)
\ No newline at end of file
+)
```

Comment:
Please don't unnecessarily change the white space. See https://gist.github.com/mherrmann/5ce21814789152c17abd91c0b3eaadca.

Reply:
I immagine this is due to my editor settings.